src/Concerns/IslamicHolidays.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Spatie\Holidays\Concerns;
+
+use Carbon\CarbonImmutable;
+
+trait IslamicHolidays
```

Comment:
We already have an IslamicCalendar trait which you can use

Reply:
This trits depends on the constants variable and I explained the issue in the above replay.